mobile/src/main/java/ch/epfl/sdp/mobile/application/authentication/AuthenticatedUser.kt

```diff
@@ -72,7 +74,7 @@ class AuthenticatedUser(
    * @param unfollowed the [Profile] to unfollow.
    */
   suspend fun unfollow(unfollowed: Profile) {
-    firestore.collection("users").document(unfollowed.uid).update {
+    firestore.collection(ProfileDocument.Collection).document(unfollowed.uid).update {
       arrayRemove("followers", user.uid)
```

Comment:
Missing field.

Reply:
Added in 715a39794e4fd9228eaaf11ad0bf8a1682f5edfd